Dockerfile

```diff
@@ -0,0 +1,37 @@
+# choose the Image which has Node installed already
+FROM node:16 AS build
+
+# add arg for the network
+ARG NETWORK
+
+RUN echo ${NETWORK}
+
+# make the 'app' folder the current working directory
+WORKDIR /app
+
+# copy package.json to cache the np latter on
+COPY package.json .
```

Comment:
why coping the package.json and then copy the whole project? it will be copied with the project